compute/admin_guide/continuous_integration/jenkins_pipeline_project.adoc

```diff
@@ -188,6 +190,7 @@ pipeline {
                 key: '',
                 logLevel: 'info',
                 podmanPath: '',
+                // The project field below is only applicable if you are using Prisma Cloud Compute Edition and have set up projects (multiple consoles) on Prisma Cloud. This field is not the name of the Jenkins pipeline.
```

Comment:
I think this is enough "The project field below is only applicable if you are using Prisma Cloud Compute Edition and have set up projects (multiple consoles) on Prisma Cloud." The second sentence will just confused them